cmd/certsuite/generate/catalog/catalog.go

```diff
@@ -138,7 +137,7 @@ func addPreflightTestsToCatalog() {
 	// Create artifacts handler
 	artifactsWriter, err := artifacts.NewMapWriter()
 	if err != nil {
-		logrus.Errorf("error creating artifact, failed to add preflight tests to catalog")
+		log.Error("error creating artifact, failed to add preflight tests to catalog")
```

Comment:
Please capitalize the first word's letter in the message. Also, include the error in the message like:
```
log.Error("Error creating artifacts catalog writer for preflight tests: %v", err)
```

Same for the other changes.

Reply:
Done.  I found some other instances of it as well.